docs/configuration/inter_node_data_path.md

```diff
@@ -137,3 +137,33 @@ paths = arch.search_paths(
     node_filter = node_filter,
     communication_filter = comm_filter)
 ```
+
+### Combine paths
+
+If the path from `source node` to `destination node` is too long, `Architecture.search_paths()` takes a long time and may find many paths that don't need to be considered.
+It may be laborious to select a correct target path.
+`Architecture.combine_path()` combines paths which are found by `Architecture.search_paths()`.
+By searching short paths and combining them repeatedly, you can get a target path (Ex. from head node to tail node of a target application).
+
+`Architecture.combine_path()` can combine when the connect nodes, which are the tail node of the first path and the head node of the second path, are same.
```

Comment:
I think it is not necessary to describe the conditions of the exception in detail.

```suggestion
Usage of `Architecture.combine_path()` is as following.
```

Reply:
I modified as suggestion.